docs/reference/ilm/actions/ilm-rollover.asciidoc

```diff
@@ -81,6 +81,14 @@ replicas are ignored.
 TIP: To see the current shard size, use the <<cat-shards, _cat shards>> API.
 The `store` value shows the size each shard, and `prirep` indicates whether a
 shard is a primary (`p`) or a replica (`r`).
+
+`max_shard_docs`::
```

Comment:
Maybe a better name would be max_single_shard_docs? 
Since this condition will apply if any shard reach this limit?

Reply:
> Since this condition will apply if any shard reach this limit?
yeah, any shard that reaches the limit will cause a rollover.

I'm not sure, which one of the following is better:
- max_shard_docs
- max_single_shard_docs
- max_primary_shard_docs

We can discuss and make a decision.

Reply:
Maybe `max_primary_shard_docs ` is better here? Given how the doc count is computed in `TransportRolloverAction#buildStats(...)`. In this code non primary shards are filtered, just like this
code is already doing when computing `maxPrimaryShardSize`.

Reply:
yeah, I also prefer `max_primary_shard_docs`, I will change the name to `max_primary_shard_docs `